symbulate/results.py

```diff
@@ -10,7 +10,6 @@
 import pandas as pd
```

Comment:
I completely missed this import that you added in a previous commit. I really would like to avoid importing Pandas if possible. It's a large library with a huge overhead. Can you try to do everything in Matplotlib?